crates/extensions/c8y_mapper_ext/src/converter.rs

```diff
@@ -746,7 +746,9 @@ impl Converter for CumulocityConverter {
             .ops_dir
             .eq(&self.cfg_dir.join("operations").join("c8y"))
         {
-            add_or_remove_operation(message, &mut self.operations)?;
+            // Re populate the operations irrespective add/remove/modify event
+            self.operations = get_operations(message.ops_dir.clone())?;
+            dbg!(&self.operations);
```

Comment:
Don't forget to remove that before the final commit.

Reply:
Still to be removed ;-)

Reply:
done